build-platform.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+
+# Default RPC URL
+DEFAULT_RPC_URL="http://localhost:8545"
+
+# Function to start the settlement layer
+start_settlement_layer() {
+    cd ~/
+    git clone git@github.com:primevprotocol/go-ethereum.git
```

Comment:
Cloning geth from top level like this defeats the purpose of having the docker setup live within geth repo, maybe suggests monorepo would be appropriate in medium/long term

Reply:
@shaspitz Agreed. However, this might be ok for now until we have the artifact repository where we can push all our binaries.